.eslintrc.js

```diff
@@ -50,7 +57,20 @@ module.exports = {
     'jest': true,
     'expect': true
   },
+  settings: {
+    jsdoc: {
+      ignorePrivate: true,
+      publicFunctionsOnly: true
+    }
+  },
   rules: {
+    'jsdoc/require-description-complete-sentence': [ERROR, {abbreviations: ['e.g', 'etc']}],
```

Comment:
should this have i.e too?

Reply:
are you seeing an example where something was mis-capitalized? I don't want to add a ton of them from the get go, i figure as people use them, they can add them here